docs/reference/commandline/ps.md

```diff
@@ -27,6 +27,7 @@ Options:
                         - since=(<container-name>|<container-id>)
                         - ancestor=(<image-name>[:tag]|<image-id>|<image@digest>)
                           containers created from an image or a descendant.
+                        - is-task=(true|false|any)
```

Comment:
@stevvooe We use it for `docker search` (is-official, is-automated); https://docs.docker.com/engine/reference/commandline/search/#/filtering, possibly other locations that I cannot recall right now :smile: